package.json

```diff
@@ -13,7 +13,7 @@
     "copy": "npm run copy:gov && npm run copy:assets",
     "copy:gov": "sh ./bin/copy-gov && node ./bin/fix-layout.js",
     "copy:assets": "sh ./bin/copy-assets",
-    "build:js": "sh ./bin/build-js && npm run build-map",
+    "build:js": "rm dist/* -f && webpack",
```

Comment:
This means copy_gov and copy_assets (see above) wont get called

Will this matter, is it covered elsewhere?
If they aren't required the whole bin script ought to go.